docs/components/Intro.js

```diff
@@ -270,11 +270,6 @@ const cloudInfo = {
       path: "/cloud/tcld",
       name: "tcld",
     },
-    {
-      exit: true,
-      path: "/cloud/release-notes",
-      name: "Release notes",
-    },
```

Comment:
I could make a _pro forma_ argument for keeping this link and targeting the new location, but I'm not gonna argue strongly for it.